deepchem/models/torch_models/cgcnn.py

```diff
@@ -221,19 +223,44 @@ def forward(self, dgl_graph):
       If mode == 'classification', the shape is `(batch_size, n_tasks, n_classes)` (n_tasks > 1)
       or `(batch_size, n_classes)` (n_tasks == 1) and the output values are probabilities of each class label.
     """
-    graph = dgl_graph
-    # embedding node features
-    node_feats = graph.ndata.pop('x')
-    edge_feats = graph.edata.pop('edge_attr')
-    node_feats = self.embedding(node_feats)
+    
+
+    if type(dgl_graph) == list:
+      graph = self.graph
+      node_feats = dgl_graph[0]
+      edge_feats = dgl_graph[1]
+      print('Input is list!')
+    else:
+      graph = dgl_graph
+      node_feats = graph.ndata.pop('x')
+      edge_feats = graph.edata.pop('edge_attr')
 
+    
+    node_feats = self.embedding(node_feats)
+    
     # convolutional layer
     for conv in self.conv_layers:
       node_feats = conv(graph, node_feats, edge_feats)
 
     # pooling
     graph.ndata['updated_x'] = node_feats
-    graph_feat = F.softplus(self.pooling(graph, 'updated_x'))
+
+    pool = self.pooling(graph, 'updated_x')
```

Comment:
Can you please comment these changes?

Reply:
I don't use  `pool` variable for the model, but just to compare the results.